Cannot build Mockito on Windows 64b with Java 8

Hi guys,

 I am trying to build Mockito on my Windows 64 machine with jdk1.8.0_66 and the build is failing. I tried the `master `branch and also the `release/2.x` branch, both are having the same problem. 

When I run `gradlew clean build` I am getting:

```
mockito\src\test\java\org\mockitousage\matchers\MatchersTest.java:599: error: unclosed character literal
        mock.oneArg(Character.valueOf('Ôé¼'));
                                      ^
mockito\src\test\java\org\mockitousage\matchers\MatchersTest.java:599: error: illegal character: '\u201a'
        mock.oneArg(Character.valueOf('Ôé¼'));
                                        ^
mockito\src\test\java\org\mockitousage\matchers\MatchersTest.java:599: error: illegal character: '\u00ac'
        mock.oneArg(Character.valueOf('Ôé¼'));
                                         ^
mockito\src\test\java\org\mockitousage\matchers\MatchersTest.java:599: error: unclosed character literal
        mock.oneArg(Character.valueOf('Ôé¼'));
                                          ^
mockito\src\test\java\org\mockitousage\matchers\MatchersTest.java:599: error: ';' expected
        mock.oneArg(Character.valueOf('Ôé¼'));
                                            ^
5 errors
:compileTestJava FAILED
:removeTestResources

FAILURE: Build failed with an exception.
```


After removing the line containing `        mock.oneArg(Character.valueOf('€'));` inside **MatchersTest.java** on line 599, the build works OK.

Anyone else facing similar trouble?

--- a/build.gradle
+++ b/build.gradle
@@ -39,6 +39,7 @@ allprojects {
     tasks.withType(JavaCompile) {
         //I don't believe those warnings add value given modern IDEs
         options.warnings = false
+        options.encoding = 'UTF-8'
     }
 }
 

